features/admin-file-attributes.feature

```diff
@@ -0,0 +1,142 @@
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @published
+  Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons
+    Given I sign in as an admin
+    When I am on "/adminFile/update/id/13973"
+    Then I should see a button "New Attribute"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Edit"
+    And I should see a button input "Delete"
+    And I should see a button input "Save"
+
+  @ok @javascript @published
+  Scenario:  Go to a published dataset found in production-like database, should see the File Attribute Id of the last modified attribute
+    Given I sign in as an admin
+    When I go to "/adminFile/view/id/13973"
+    Then I should see field "File Attribute Id" with value "1 "
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    Then I should not see "last_modified"
+    And I should not see "2013-7-15"
+    And I should not see a button "Delete"
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute and save
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    And I press "Save"
+    Then I am on "/adminFile/view/id/13973"
+    And I should see field "File Attribute Id" with empty value
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute and check for history tab
+    Given I sign in as an admin
+    And I go to "/adminFile/update/id/13973"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Delete"
+    When I press "Delete"
+    And I press "Save"
+    Then I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    And I should see "History" tab with text "Termitomyces_assembly_v1.0.fa.gz: file attribute deleted"
+
+  @ok @javascript @nonPublished
+  Scenario: Go to a non published dataset found in production-like database, should see a keyword attribute and a camera parameters attribute and their File Attribute Id
```

Comment:
Scenario could be `Display attributes in non-published dataset`. I would think about separating into 2 scenarios, one scenario to check display of attributes on file update page and second scenario to check display for file attributes on file view page.